drqa/tokenizers/spacy_tokenizer.py

```diff
@@ -25,19 +25,19 @@ def __init__(self, **kwargs):
         model = kwargs.get('model', 'en')
         self.annotators = copy.deepcopy(kwargs.get('annotators', set()))
         nlp_kwargs = {'parser': False}
-        if not {'lemma', 'pos', 'ner'} in self.annotators:
+        if ('lemma' not in self.annotators) and ('pos' not in self.annotators) and ('ner' not in self.annotators):
```

Comment:
`if not any([p in self.annotators for p in ['lemma', 'pos', 'ner']):`

Reply:
(or `if len({'lemma', 'pos', 'ner'} & self.annotators) == 0`)

Reply:
More simple and graceful.